### ISSUE TITLE
Layout of default keyboard shortcuts should not be dependent on keyboard layout

### ISSUE THREAD

**Issue by finwe** [musescore/MuseScore#19619]
### Your idea

Given that basic note durations do not react to numeric block keys pressed, I guess they are meant to be selected with the upper keyboard row with numbers.

I believe the default shortcuts should take current keyboard layout into consideration so that no matter the layout selected, the physical shortcut is always the same.

### Problem to be solved

When eg. a Czech keyboard layout is selected, the upper row is primary entering accented characters `ěščřžýáíé`, numbers are entered with shift. But the note durations react not on keys, but on specific characters, so I can select durations only with shift pressed, which is a bit annoying and unnecessary.

Also, when changing the keyboard shortcuts in settings, the numeric block acts the same as the upper row of numbers (just changes the number), but then doesn't react anyway, which is confusing. This might actually be worth of a separate issue?

**Comment by mmeyn** [comment]
I disagree: I use the [Neo layout](https://neo-layout.org/) where you can enter numbers (on “layer 4”) without having to stretch to the upper row and I don’t want to miss that. However I have to say that not all default shortcuts work with Neo layout on my system: f. e. Up and Down (= “layer 4” of qwerty <kbd>E</kbd> and <kbd>D</kbd>) work, <kbd>Ctrl</kbd>+Up also works but <kbd>Ctrl</kbd>+Down works only with the physical <kbd>↓</kbd> key.

**Comment by finwe** [comment]
@Jojo-Schmitz Well, it doesn't for me. So maybe this is rather a bug report?

The point of this issue though is that sometimes, depending on the keyboard layout, the duration using the top numbers row of the keyboard can be harder to access, which is uncomfortable.

**Comment by Jojo-Schmitz** [comment]
It might work for me due to my 'hand-made' shortcuts.xml.
